assessments/guide/security-reviewer.md

```diff
@@ -12,20 +12,28 @@ involving deployment scenarios and the impact of attacks.
 
 ## Qualifications
 
+### Required
+
 Unless approved by the SIG chairs, at least one of the reviewers will
-have previously performed a SAFE WG audit.  (Exemptions are expected to be
-granted bootstrap the process but only in extreme cases later on.)  In
-general, a reviewer should have performed security audits for diverse
-organizations.  The ideal reviewer should also have been the recipient
-of security audits for a software project they manage.  Note that it is
-encouraged to have participation (shadowing) from participants that are not
-yet qualified to help them gain the necessary skills to be a reviewer
+have previously performed a CNCF project security assessment.
+Exemptions to this are reviewed case by case upon established need by the CNCF Security SIG Chairs and Co-Chairs in order to bootstrap the process as appropriate.
```

Comment:
I think this could just say "CNCF Security SIG Chairs" (and co-chairs is implicit) -- we only have a single chair role

Reply:
corrected